src/common/options/mds-client.yaml.in

```diff
@@ -390,6 +390,31 @@ options:
   default: true
   services:
   - mds_client
+- name: fuse_entry_timeout
+  type: int
+  level: advanced
+  desc: Validity timeout (in seconds) for the name
+  long_desc: If directory entries are changed/deleted only as a result of requests 
+    that come through the kernel, this should be set to a very large value.
```

Comment:
@xieshengGZ This desc is not precise here. Please fix it.

Something like:

`This will tell kernel how long could the dentry be cached in dcache, once enabled this should be set to a very large value.`

Reply:
Well！